renovate.json

```diff
@@ -1,15 +1,52 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["github>konflux-ci/mintmaker//config/renovate/renovate.json"],
+  "extends": [
+    "github>konflux-ci/mintmaker//config/renovate/renovate.json",
+    ":dependencyDashboard"
+  ],
   "timezone": "America/New_York",
   "schedule": ["before 10pm"],
-  "enabledManagers": ["asdf", "tekton", "github-actions"],
+  "enabledManagers": ["asdf", "tekton", "github-actions", "gomod"],
   "baseBranches": ["main", "release-v0.5", "release-v0.6"],
   "github-actions": {
     "enabled": true,
-    "schedule": ["before 10pm"],
     "separateMinorPatch": true,
+    "schedule": ["0 15 * * 0"],
     "groupName": "GitHub Actions Updates"
-  }
+  },
+  "packageRules": [
+    {
+      "matchManagers": ["gomod"],
+      "matchPackageNames": ["go"],
+      "enabled": false
+    },
+    {
+      "matchManagers": ["gomod"],
+      "matchPaths": ["/"],
```

Comment:
(Maybe for later, but) how about putting all the matchPaths in one list, and combining the four groups?

Reply:
Let's see how it looks? Maybe I'm being overcautious and it'll be fine, but I didn't know if we should have them all together.